testing/utils/openshift_ci/pr_args.sh

```diff
@@ -90,7 +90,7 @@ else
 fi
 
 REQUIRED_AUTHOR=$(jq -r .user.login <<< "$pr_json")
-REQUIRED_AUTHOR_ASSOCIATION=CONTRIBUTOR
+REQUIRED_AUTHOR_ASSOCIATION=COLLABORATOR
```

Comment:
you should be good to drop this change now that Charan's [PR](https://github.com/openshift-psap/topsail/pull/362) is merged